<launch>
	<rosparam command="load" file="$(find cr_control)/config/control.yaml"/>
	<rosparam command="load" file="$(find cr_control)/config/wheel_hwin_settings.yaml"/>
	<!-- Launch hardware interface then add the controller manager and controllers-->
	<include file="$(find vicon_bridge)/launch/vicon.launch" >
		<arg name="datastream_hostport" value="192.168.0.151" />
	</include>
	<node name="collect_data_node" pkg="cr_control" type="collect_data.py" output="screen"/>
	
	

	<node name="wheel_hardware_interface" 
		pkg="cr_control" 
		type="wheel_hardware_interface" 
		output="screen"/>
	<node name="controller_manager_node" 
		pkg="controller_manager"
		type="spawner"  
		output="screen"  
		args="cr_velocity_controller cr_joint_publisher" />
</launch>
